policyengine_us/parameters/gov/states/ny/tax/income/main/head_of_household.yaml

```diff
@@ -27,30 +32,42 @@ brackets:
     rate:
       2021-01-01: 0.059
       2022-01-01: 0.0585
+      2023-01-01: 0.055
   - threshold:
       2021-01-01: 32_200
+      2023-01-01: 107_650
```

Comment:
@vrathi101 could you make the brackets align with the forms / legal code and if the number of brackets does not match -> make the bottom rate .inf